desc/objectives/_coils.py

```diff
@@ -1182,8 +1198,23 @@ def __init__(
         self._eq = eq
         self._field = field
         self._field_grid = field_grid
-        self._vacuum = vacuum
-        things = [field]
+        self._vacuum = vacuum or qfm_surface
```

Comment:
might be better to throw an error here if they're incompatible?

Reply:
The thing is, by default `vacuum=False`, so an error would be thrown if `qfm_surface=True` is passed in.

I guess that is fine if we want users to have to be verbose about asking for qfm surface and specifying they are for a vacuum field